test/did-jwt-vc.test.ts

```diff
@@ -1,93 +1,114 @@
+   * Private key: 5JibpxxNpkqdejc38KD9xZF3fKHtTUonKCAYiZbwNPsgoKbw6FQ
+   */
+  const publicKeys = [
+    'EOS6Q8zrGD6dceAiRx1FxruYYCZaoAr4cintr5mpibHQC6ToxUreA',
+    'EOS7EEmiZSRfqMmVse9bycExAAZ1ixUioPavkGTbotiGLryhACufQ',
+    'EOS7y23bQGv367BFcv4eKrFdq6BTZsTh4gbDziNdRiffv1tq6DnaP',
+  ];
+  const privatekeys = [
+    '5JAJ7BfYKdRnrSQCsdcBqrCcBVQQSuQ77fuRAJ5fcbQ3UDhuLLZ',
+    '5JcKy5rAp4rTnE9za1CNm5xBG4DnJ2T29cYpz87kRVRdQqv1K8x',
+    '5JibpxxNpkqdejc38KD9xZF3fKHtTUonKCAYiZbwNPsgoKbw6FQ',
+  ];
+  const Resolver = {
+    resolve: async (did: string) => {
+      const parsed = parse(did);
+      if (!parsed) throw new Error('could not parse did');
+      const methodId = checkDID(parsed, antelopeChainRegistry);
+      if (!methodId) throw new Error('invalid did');
+      const account = await fetchAccount(methodId, did, parsed, { fetch });
+      account.permissions[0].required_auth.threshold = 2;
+      account.permissions[0].required_auth.keys = publicKeys.map(key => {
+        return { key, weight: 1 };
+      });
+      const didDoc = createDIDDocument(methodId, did, account);
+      return {
+        didResolutionMetadata: {},
+        didDocument: didDoc,
+        didDocumentMetadata: {},
+      };
+    },
+  };
   it('Issues a simple Antelope credential signed by one key', async () => {
-    const did = "did:eosio:eos:testnet:jungle:reball1block";
+    const did = 'did:eosio:eos:testnet:jungle:reball1block';
 
     const keyIssuer1: Issuer = {
-      did: did + "#active-0",
-      signer: createSigner(PrivateKey.from("5KSKD681YRwQjwDkr8TLkUUU5adHy1CWGuLiow1DR5ToZF5oiUQ")),
-      alg: 'ES256K-R'
-    }
+      did: did + '#active-0',
+      signer: createSigner(
+        PrivateKey.from('5KSKD681YRwQjwDkr8TLkUUU5adHy1CWGuLiow1DR5ToZF5oiUQ')
+      ),
+      alg: 'ES256K-R',
+    };
 
     const vcPayload: JwtCredentialPayload = {
       sub: did,
       nbf: 1562950282,
       vc: {
         '@context': ['https://www.w3.org/2018/credentials/v1'],
-        type: ['VerifiableCredential'],
+        type: ['ConditionalProof2022'],
         credentialSubject: {
           degree: {
             type: 'BachelorDegree',
-            name: 'Baccalauréat en musiques numériques'
-          }
-        }
-      }
-    }
+            name: 'Baccalauréat en musiques numériques',
+          },
+        },
+      },
+    };
 
     const vcJwt = await createVerifiableCredentialJwt(vcPayload, keyIssuer1);
     const decodedJwt = decodeJWT(vcJwt);
-    console.log(vcJwt);
-    const antelopeDID = new AntelopeDID({ fetch })
-    const resolver = { resolve: antelopeDID.resolve.bind(antelopeDID)  };
-    console.log(JSON.stringify(await resolver.resolve(did , { accept: 'application/did+ld+json' })));
-    
-    expect(verifyCredential(vcJwt,resolver )).resolves.toBeTruthy();
-    expect(decodedJwt).toBeDefined();
-  })
+    const antelopeDID = new AntelopeDID({ fetch });
+    const resolver = { resolve: antelopeDID.resolve.bind(antelopeDID) };
```

Comment:
shouldn't this be?
```ts
const resolver = { resolve: antelopeDID.resolve.bind(Resolver.resolve) };
```

Reply:
i am binding the antelopeDID class so that the properties get passed to the resolver + here i am using the antelope resolver not the mock resolver because there is no need to use the mock here and also i wanted to give an example on how to use the real resolver

Reply:
and test the real one